### ISSUE TITLE
NameError: name 'basestring' is not defined

### ISSUE THREAD

**Issue by tranvnhan** [hellochick/Indoor-segmentation#6]
Hi.
I ran the inference using the command:
`python inference.py --img_path=test_img.png --restore_from=/restore_weights/`

but the error came out:

>   File "/home/nhan/git_workspace/Indoor-segmentation/network.py", line 78, in feed
>     if isinstance(fed_layer, basestring):
> NameError: name 'basestring' is not defined

I look for basestring in **network.py** and `basestring` is really not defined. 
Could you help me, please?

**Comment by chuzcjoe** [comment]
Hi, did you solve it? i have the same problem.

**Comment by tranvnhan** [comment]
> 
> 
> Hi, did you solve it? i have the same problem.

No. Problem is not resolved yet. In the mean time, I found another repository which worked pretty well. 
You can take a look [here](https://github.com/yunlongdong/FCN-pytorch-easiest).

**Comment by hellochick** [comment]
You can simply replace it with “str”.
That is an error because of different version of python.
